apps/api-server/src/routes/widget/widget.js

```diff
@@ -0,0 +1,149 @@
+const express = require('express');
+const bruteForce = require('../../middleware/brute-force');
+const db = require('../../db');
+const fs = require('fs');
+const config = require('config');
+const path = require('path');
+
+let router = express.Router({ mergeParams: true });
+
+// brute force
+router.use(bruteForce.globalMiddleware);
+
+router
+  .route('/:widgetId([a-zA-Z0-9]+)')
+  .all((req, res, next) => {
+    if (!req.params?.widgetId) {
+      return next();
```

Comment:
Er wordt nu geen foutmelding gegeven bij een route die wel een id verwacht. i.p.v. return next zou ik daar ook een badrequest error of iets dergelijks verwachten

Reply:
Yes, heb ik nu toegevoegd. Ook al is de check dubbelop, zonder widgetId kom je namelijk niet op de route.

Reply:
Dat is ook zo inderdaad, had ik niet bij stilgestaan!